bin/install_rasdaman.sh

```diff
@@ -46,6 +48,33 @@ setup_rasdaman_repo()
   echo "deb [arch=amd64] $rasdaman_pkgs_url/deb $codename $release" > /etc/apt/sources.list.d/rasdaman.list
 }
 
+unpack_war_file()
+{
+  # unpacks $1.war into a directory $1, and removes $1.war
+  local war_name="$1"
+  local war_file="$war_name.war"
+  pushd "$TOMCAT_WEBAPPS"
+  if [ -f "$war_file" ]; then
+    mkdir -p "$war_name"
+    mv "$war_file" "$war_name"
+    pushd "$war_name"
+    jar xf "$war_file" && rm -f "$war_file" # extract
+    popd
+  fi
+  popd
+}
+
+patch_gdal_java_jar()
+{
+  # patch the gdal-java dependency to the correct version for Lubuntu 18.04
+  # TODO: remove the below once http://rasdaman.org/ticket/2144 is fixed
+  if [ -d "$TOMCAT_WEBAPPS/rasdaman/WEB-INF/lib/" ]; then
+    rm -f $TOMCAT_WEBAPPS/rasdaman/WEB-INF/lib/gdal-1.10.*.jar
+    wget "http://kahlua.eecs.jacobs-university.de/~dmisev/gdal-2.3.0.jar" \
```

Comment:
The correct gdal version for OSGeoLive is 2.4.2
The system is based on this ppa:

https://launchpad.net/%7Eosgeolive/+archive/ubuntu/nightly/+packages?field.name_filter=&field.series_filter=bionic&field.status_filter=published&batch=75

Reply:
The gdal-java jar version used here (2.3.0) will work fine with the gdal version 2.4.2.